python/ray/tune/examples/dragonfly_example.py

```diff
@@ -17,9 +17,9 @@
 
 def objective(config):
     for i in range(config["iterations"]):
-        vol1 = config["point"][0]  # LiNO3
-        vol2 = config["point"][1]  # Li2SO4
-        vol3 = config["point"][2]  # NaClO4
+        vol1 = config["LiNO3_vol"]  # LiNO3
+        vol2 = config["Li2SO4_vol"]  # Li2SO4
+        vol3 = config["NaClO4_vol"]  # NaClO4
```

Comment:
Please note that we keep backwards compatibility here! `config["point"][0]` still works

Reply:
great!